src/styles/profile/profile.css

```
@@ -0,0 +1,46 @@
+.profile {
+  padding: 16px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 16px;
+  row-gap: 16px;
```

If you have two values which are the same, I'd recommend using the shorthand:
```suggestion
 grid-gap: 16px;
```